pkg/metrics/recorder.go

```diff
@@ -93,3 +111,9 @@ func (r *Recorder) SetVirtualServiceInactive(name string, mesh string) {
 func (r *Recorder) SetRequestDuration(kind string, object string, operation string, duration time.Duration) {
 	r.apiRequestDuration.WithLabelValues(kind, object, operation).Observe(duration.Seconds())
 }
+
+// RecordOperation records the duration of operation (e.g. API Call, Function exection)
+// based on object kind, name and operation type. The operation type can be get, create, update, delete
```

Comment:
> The operation type can be get, create, update, delete

In the instances in this PR you are using function names, do you want to make this comment reflect those usages?  Also object is always empty, when would it be used?

Reply:
> Also object is always empty, when would it be used?

object is not always empty, for e.g. 
```
pkg/aws/cloudmap.go:            c.stats.RecordOperation("cloudmap", "service", "create", time.Since(begin))
pkg/aws/cloudmap.go:            c.stats.RecordOperation("cloudmap", "service", "getByID", time.Since(begin))
```